packages/core/parcel/package.json

```diff
@@ -54,8 +54,7 @@
         "@parcel/fs": false,
         "@parcel/package-manager": false,
         "@parcel/fs-search": false,
-        "highlight.js": false,
```

Comment:
`highlight-js` makes the bundle very large. Although many of its languages are not used.

Reply:
Made a pull request to fix this issue
https://github.com/parcel-bundler/parcel/pull/6196